.github/workflows/ci.yaml

```diff
@@ -114,6 +120,17 @@ jobs:
             exit 1
           fi
       
+      - name: Determine CUDA image to use
+        id: cuda-image
+        shell: bash -x -e {0}
+        run: |
+          if [[ "${{ github.event_name }}" == "workflow_dispatch" ]]; then
```

Comment:
Not quite understand the purpose of this if-statement. The default value is `latest` anyways. Or am I missing something?